pkg/proxy/proxier.go

```diff
@@ -583,18 +586,18 @@ func (proxier *Proxier) closePortal(service string, info *serviceInfo) error {
 	return errors.NewAggregate(el)
 }
 
-func (proxier *Proxier) closeOnePortal(portalIP net.IP, portalPort int, protocol api.Protocol, proxyIP net.IP, proxyPort int, name string) []error {
+func (proxier *Proxier) closeOnePortal(portalIP net.IP, portalPort int, protocol api.Protocol, proxyIP net.IP, proxyPort int, name string, rewrite bool) []error {
```

Comment:
Given that you only ever call it with rewrite == false, why are you adding the parameter to close?